### PR TITLE
Fix nupkg naming in Travis

### PR DESCRIPTION
**Pull request by sgladkov**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -78,8 +78,8 @@ jobs:
       api_key: "$GITHUB_TOKEN1"
       file_glob: true
       file:
-      - bin/Debug/Lachain.CommunicationHub.Net.0.0.*.nupkg
-      - ../Lachain.CommunicationHub.Native.0.0.*.nupkg
+      - Lachain.CommunicationHub.Native.*nupkg
```
Comment (kunalgoyal9) [inline .travis.yml:81]: it should be `../Lachain.CommunicationHub.Native.*nupkg`

**Review by kunalgoyal9 — CHANGES_REQUESTED** [review]
path issue

File: .travis.yml (hunk 1 above)
Comment (sgladkov) [reply .travis.yml]: fixed

**Comment by kunalgoyal9** [comment]
Looks good

**Review by kunalgoyal9 — APPROVED** [review]

**Pull request merged**